packages/nodes-base/nodes/HaloPSA/GenericFunctions.ts

```diff
@@ -96,7 +96,7 @@ export async function haloPSAApiRequest(
 		return result;
 	} catch (error) {
 		const message = (error as JsonObject).message as string;
-		if (method === 'DELETE' || 'GET' || ('UPDATE' && message)) {
+		if (method === 'DELETE' || method === 'GET' || (method === 'UPDATE' && message)) {
```

Comment:
I'm guessing this was never actually tested, as this is equivalent of  `if (method === 'DELETE') {` 😅